Alarm-Clock/alarm-clock.py

```diff
@@ -0,0 +1,85 @@
+from tkinter import *
```

Comment:
- When time is up, play an alarming tone. 
- Use range function to define Hours, Minutes, and seconds instead of these hardcoded tuples

Reply:
@Anupreetadas These changes also need to be done